custom_components/octopus_intelligent/octopus_intelligent_system.py

```diff
@@ -179,7 +179,7 @@ async def start(self):
                 _LOGGER.error(f"Account {self._account_id} not found in accounts {accounts}")
                 raise Exception(f"Account {self._account_id} not found in accounts {accounts}")
         except Exception as ex:
-            _LOGGER.error(f"Authentication failed : {ex.message}. You may need to check your token or create a new app in the gardena api and use the new token.")
+            _LOGGER.error(f"Authentication failed : {ex}. You may need to check your token or create a new app in the gardena api and use the new token.")
```

Comment:
Similarly as the other comment, when `start()` is executed it revealed other issues. Here the issue was `'TypeError' object has no attribute ‘message’`:
```
2024-02-25 23:22:42.043 ERROR (MainThread) [custom_components.octopus_intelligent] Got error when setting up Octopus Intelligent Integration: 'TypeError' object has no attribute ‘message'
```

This log message and most of the `start()` function is further modified / improved in another PR: #40